lib/ansible/modules/cloud/lxd/lxd_container.py

```diff
@@ -458,7 +458,7 @@ def _needs_to_change_container_config(self, key):
         if key == 'config':
             old_configs = dict((k, v) for k, v in self.old_container_json['metadata'][key].items() if not k.startswith('volatile.'))
             for k, v in self.config['config'].items():
-                if old_configs[k] != v:
+                if k in old_configs and old_configs[k] != v:
```

Comment:
I'd use `.get()` instead of checking for key.

Reply:
Yes thatcwould be better in that case:
```if old_configs.get(k, None) != v``` but not sure if v can be None